content/blog/managing-secrets-with-pulumi/index.md

```diff
@@ -204,7 +204,7 @@ passwords or access credentials.
 You might be wondering how these values are actually encrypted. We use
 the same encryption that we have always used for our configuration
 system. This means when storing state with
-<app.pulumi.com>, we use a key managed by the <app.pulumi.com> service, specific to your stack, to
+&lt;app.pulumi.com&gt;, we use a key managed by the &lt;app.pulumi.com&gt; service, specific to your stack, to
```

Comment:
I'm guessing the intention was for these to be [automatically linked](https://github.com/russross/blackfriday/blob/41c5fccfd6f60cb5fd077118835b1d7122498df6/testdata/Markdown%20Documentation%20-%20Syntax.text#L833-L839), but perhaps the `https://` prefix is needed for that to happen?

```suggestion
<https://app.pulumi.com>, we use a key managed by the <https://app.pulumi.com> service, specific to your stack, to
```

Wondering if we make the above suggestion, if we'd see these as links as intended in the Pulumify preview.

Reply:
Ah, that makes more sense in terms of the original intent. Sure, let me swap to that.